cl_dll/ff/ff_hud_deathnotice.cpp

```diff
@@ -264,10 +265,9 @@ void CHudDeathNotice::Paint()
 		}
 		else
 		{
-
-			wchar_t victim[ 256 ];
-			wchar_t killer[ 256 ];
-
+			wchar_t victim[ 256 ];			
+			wchar_t killer[ 512 ]; // bumped this up for kill-assist msgs 

```

Comment:
Yeah the 256 thing is pretty typical of FF code--when in doubt, make large capacity strings.